### PR TITLE
Implement nbh vertex client

### PR DESCRIPTION
**Pull request by serboctor**
Resolves: https://github.com/threefoldtech/jumpscaleX/issues/786

### CODE REVIEW

File: Jumpscale/clients/nbh_vertex/NBHClient.py [hunk 1]
```diff
@@ -0,0 +1,424 @@
+import requests
+import json
```
Comment (xmonader) [inline Jumpscale/clients/nbh_vertex/NBHClient.py]: can we move to using j.data.serializers.json through the client?

File: Jumpscale/clients/nbh_vertex/NBHClient.py [hunk 2]
```diff
@@ -0,0 +1,424 @@
+import requests
```
Comment (xmonader) [inline Jumpscale/clients/nbh_vertex/NBHClient.py:1]: i think jumpscale wraps it using j.clients.http

File: Jumpscale/clients/nbh_vertex/NBHClient.py [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+            return m
+    return ""
+
+
+def raise_if_error(data):
+    error_string = ""
+    if isinstance(data, str) or isinstance(data, int):
+        error_string = get_error_string(str(data))
+        if error_string:
+            raise Exception(error_string)
```
Comment (xmonader) [inline Jumpscale/clients/nbh_vertex/NBHClient.py]: let's use j.exceptions.* types

File: Jumpscale/clients/nbh_vertex/NBHClient.py (hunk 2 above)
Comment (serboctor) [reply Jumpscale/clients/nbh_vertex/NBHClient.py:1]: I don't see anything there regarding sessions

**Pull request merged**